druid/src/test/java/org/apache/calcite/test/DruidAdapterIT.java

```diff
@@ -755,28 +779,16 @@ private void checkGroupBySingleSortLimit(boolean approx) {
   /** Tests a query that contains no GROUP BY and is therefore executed as a
    * Druid "select" query. */
   @Test public void testFilterSortDesc() {
-    final String sql = "select * from \"foodmart\"\n"
```

Comment:
Why is ```select *...``` not working?
I would expect it to work as usual but to project out automatically those columns that are virtual (e.g. complex metrics). Is this the case? We would not want to fail in these cases.